.circleci/config.yml

```diff
@@ -1,36 +1,27 @@
-version: 2
+orbs:
+  shellcheck: circleci/shellcheck@2.2.2
+  docker: circleci/docker@1.5.0
+version: 2.1
 jobs:
-  lint-dockerfile:
-    machine:
-      image: ubuntu-1604:201903-01
-    steps:
-      - checkout
-      - run: wget https://github.com/hadolint/hadolint/releases/download/v1.17.2/hadolint-Linux-x86_64 -O hadolint  # yamllint disable-line rule:line-length
-      - run: chmod +x hadolint
-      - run: ./hadolint $(find . -name '*Dockerfile*')
-  lint-yaml:
-    machine:
-      image: ubuntu-1604:201903-01
+  yamllint:
+    docker:
+      - image: cimg/python:3.9.1
     steps:
       - checkout
       - run: pip install yamllint
       - run: yamllint -d .yamllint.yml .
-  lint-shell:
-    machine:
-      image: ubuntu-1604:201903-01
+  shellcheck:
+    docker:
+      - image: cimg/base:2021.02
     steps:
       - checkout
-      - run: sudo apt-get update
-      - run: sudo apt-get install shellcheck
-      - run: shellcheck *.sh
+      - shellcheck/install
+      - shellcheck/check
 workflows:
   version: 2
-  lint-dockerfile:
-    jobs:
-      - lint-dockerfile
-  lint-yaml:
-    jobs:
-      - lint-yaml
-  lint-shell:
+  linters:
     jobs:
-      - lint-shell
```

Comment:
is it possible to split it into 3 checks? that way it'll be easier to understand in Github what failed

Reply:
Done